### PR TITLE
Game Cards

### PR DESCRIPTION
**Pull request by 0marA**
Creates a play screen with cards that the player can cycle through.

### CODE REVIEW

File: client/src/pages/Home.js [hunk 1]
```diff
@@ -12,10 +12,10 @@ const Home = () => {
       </h4>
 
       <div className="home-button-container">
-        <Button href="/create" className="home-big-button">
+        <Button href="/create" className="home-big-button" style ={{display: "flex"}}>
```
Comment (Juicestus) [inline client/src/pages/Home.js:15]: Why not add to CSS?

File: client/src/pages/Play.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import { Carousel, Button } from "react-bootstrap";
+import Card from "../components/Card";
+import { useState, useEffect } from "react";
+import { NavLink } from "react-router-dom";
+
+const Play = () => {
+  const [answerCards, setAnswerCards] = useState([
+    "7 dollars and a Popeyes biscuit",
+    "The 3/5ths Compromise",
```
Comment (Juicestus) [inline client/src/pages/Play.js:9]: This is just for testing right?

**Review by Juicestus — APPROVED** [review]
LGTM👍 

Nice work, will make some improvements off this point.

File: client/src/pages/Play.js (hunk 2 above)
Comment (0marA) [reply client/src/pages/Play.js:9]: Yes

**Pull request merged**